users/v1/scripts/main.py

```diff
@@ -0,0 +1,33 @@
+import os
+import csv
+
+import global_variables
+import generate_combos
+import generate_cycles
+import generate_teacher
+
+def valid_combo(combo):
+    if (len(combo) > 0 and combo[0][0] != "#"):
+        return True
+    else:
+        return False
```

Comment:
Anti-pattern here :P
```suggestion
    return len(combo) > 0 and combo[0][0] != "#"
```

And, FYI, doing `if data_structure` in Python is (at least in 99% of the cases) equivalent to `if len(data_structure) > 0`